fastly/s3.go

```diff
@@ -101,6 +101,7 @@ type CreateS3Input struct {
 	ResponseCondition string       `form:"response_condition,omitempty"`
 	TimestampFormat   string       `form:"timestamp_format,omitempty"`
 	Redundancy        S3Redundancy `form:"redundancy,omitempty"`
+        Placement         string       `form:"placement,omitempty"`
```

Comment:
I'm sure gofmt will be angry about this, but if it's not, could you align with the line above?

Reply:
I really did try, I'm not sure why the diff is showing that.
https://github.com/RexBelli/go-fastly/blob/fd99546ee9ec7afbac829ca7fe81c087707b2377/fastly/s3.go#L104

Reply:
weird github display issue!